NBitcoin/Block.cs

```diff
@@ -236,7 +236,7 @@ public virtual uint256 GetPoWHash()
 			return GetHash();
 		}
 
-		public uint256 GetHash()
+		public virtual uint256 GetHash()

```

Comment:
why? you can override `CreateHashStream`

Reply:
I believe the Stratis code was overriding GetHash(), I'll look and see if I can do it via GetHashStream instead.